.github/workflows/test.yml

```diff
@@ -9,7 +9,7 @@ on:
       - "*"
 
 env:
-  RELAYER_VERSION: v0.2.2
+  RELAYER_VERSION: v0.2.3
```

Comment:
can we also update the other references to relayer version `v0.2.2.` in docker run/test

Reply:
Good catch 👍